docs/developer/decision-records/2025-04-25-agreement-policy-scope/README.md

```diff
@@ -0,0 +1,77 @@
+# Contract Agreement policy scope
+
+## Decision
+
+We will add a `contract.agreement` policy scope, that will be evaluated by the provider before the automatic agreement.
+
+## Rationale
+
+The negotiation flow is at the moment an automatic one: a negotiation can happen if the `contract.negotiation` policy 
+scope evaluation succeeds, or it gets terminated if the evaluation fails.
+
+To implement "manual negotiation approval", the `PendingGuard` service can be used, but this doesn't permit to
```

Comment:
looks like the last part of the sentence is missing 😄

Reply:
whops :sweat_smile: